Source/WebKit/WebProcess/cocoa/HandleXPCEndpointMessages.mm

```diff
@@ -48,6 +57,21 @@ void handleXPCEndpointMessages(xpc_object_t event, const char* messageName)
         return;
     }
 #endif
+
+#if USE(EXTENSIONKIT)
+    if (!strcmp(messageName, layerHostingContextChangedMessage)) {
+        callOnMainRunLoop([event = OSObjectPtr<xpc_object_t>(event)] () {
+            auto identifier = xpc_dictionary_get_uint64(event.get(), mediaplayerIdentifierKey);
+            auto width = xpc_dictionary_get_uint64(event.get(), presentationSizeWidthKey);
+            auto height = xpc_dictionary_get_uint64(event.get(), presentationSizeHeightKey);
+            WeakPtr<MediaPlayerPrivateRemote> player = WebProcess::singleton().remoteMediaPlayerManager().mediaPlayerFromIdentifier(WebCore::MediaPlayerIdentifier(identifier));
```

Comment:
Could this `WeakPtr` declaration just be `auto`?

Reply:
Fixed!